ui/admin/app/routes/scopes/scope/sessions.js

```diff
@@ -12,7 +12,7 @@ import runEvery from 'ember-pollster/decorators/route/run-every';
 import { notifySuccess, notifyError } from 'core/decorators/notify';
 import config from '../../../config/environment';
 import { resourceFilter } from 'core/decorators/resource-filter';
-
+import { sortBy } from 'lodash';
```

Comment:
```suggestion
import sortBy from 'lodash/sortBy';
```

We really shouldn't use these type of imports since it will pull in all of lodash. The normal lodash package can't be tree shaked unless you use `lodash-es` so we'll have the entire package in our builds. I haven't looked into how our app is built and whether it would get tree shaked correctly either so we should just stick with individual imports. 

We _could_ use the [lodash babel plugin](https://github.com/lodash/babel-plugin-lodash) but honestly we're probably not using too many lodash functions so it would be just as easy to do the individual imports.

Reply:
ahh, I didn't know that lodash isn't tree shakable by default.